src/uk/gov/hmcts/contino/azure/Acr.groovy

```diff
@@ -100,4 +93,23 @@ class Acr extends Az {
     return host?.trim()
   }
 
+  /**
+   * Retags an image in the registry with an appended hash
+   *
+   * e.g.: <image-name>:latest will also be tagged as <image-name>:latest-dfb02
+   *
+   * @param hash
+   *   a string hash (use only alphanumeric characters)
+   *
+   * @param dockerImage
+   *   the docker image to build
+   *
+   * @return
+   *   stdout of the step
+   */
+  def retagWithAppendedHash(String hash, DockerImage dockerImage) {
```

Comment:
I would rename this method to something like just `retagImage` - the tag can be anything

Reply:
OK that was the initial name I gave but I though this loooong method name was more explicit about what it it actually doing.

Reply:
since you're passing in the value the short name is fine
perhaps:
```suggestion
  def tag(String suffix, DockerImage dockerImage) {
```